<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="328" started-at="2018-10-30T06:56:07Z" finished-at="2018-10-30T06:56:08Z">
    <groups>
      <group name="Car">
        <method signature="HomePageTest.clickOnCreateOrder()[pri:1, instance:com.torenzo.qa.testcases.HomePageTest@782663d3]" name="clickOnCreateOrder" class="com.torenzo.qa.testcases.HomePageTest"/>
      </group> <!-- Car -->
    </groups>
    <test name="Default test" duration-ms="328" started-at="2018-10-30T06:56:07Z" finished-at="2018-10-30T06:56:08Z">
      <class name="com.torenzo.qa.testcases.HomePageTest">
        <test-method status="PASS" signature="loginApp()[pri:0, instance:com.torenzo.qa.testcases.HomePageTest@782663d3]" name="loginApp" is-config="true" duration-ms="10" started-at="2018-10-30T06:56:07Z" finished-at="2018-10-30T06:56:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method status="FAIL" signature="clickOnCreateOrder()[pri:1, instance:com.torenzo.qa.testcases.HomePageTest@782663d3]" name="clickOnCreateOrder" duration-ms="296" started-at="2018-10-30T06:56:07Z" finished-at="2018-10-30T06:56:08Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at io.appium.java_client.pagefactory.AppiumElementLocator.changeImplicitlyWaitTimeOut(AppiumElementLocator.java:81)
at io.appium.java_client.pagefactory.AppiumElementLocator.waitFor(AppiumElementLocator.java:99)
at io.appium.java_client.pagefactory.AppiumElementLocator.findElement(AppiumElementLocator.java:112)
at io.appium.java_client.pagefactory.interceptors.InterceptorOfASingleElement.intercept(InterceptorOfASingleElement.java:61)
at org.openqa.selenium.remote.RemoteWebElement$$EnhancerByCGLIB$$dd998978.click(<generated>)
at com.torenzo.qa.pages.HomePage.clickNewOrderCreateBtn(HomePage.java:110)
at com.torenzo.qa.testcases.HomePageTest.clickOnCreateOrder(HomePageTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnCreateOrder -->
      </class> <!-- com.torenzo.qa.testcases.HomePageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
